caffe2/serialize/inline_container.cc

```diff
@@ -232,44 +232,73 @@ PyTorchStreamReader::~PyTorchStreamReader() {
   valid("closing reader for archive ", archive_name_.c_str());
 }
 
-size_t ostream_write_func(void *pOpaque, mz_uint64 file_ofs, const void *pBuf, size_t n) {
```

Comment:
I'm not going to block on this, but it's preferable to separate unrelated formatting changes into their own diffs.